scripts/api/data/metadatablocks/astrophysics.tsv

```diff
@@ -2,13 +2,13 @@
 	astrophysics		Astronomy and Astrophysics Metadata													
 #datasetField	name	title	description	watermark	 fieldType	displayOrder	displayFormat	advancedSearchField	allowControlledVocabulary	allowmultiples	facetable	displayoncreate	required	parent	metadatablock_id
 	astroType	Type	The nature or genre of the content of the files in the dataset.		text	0		TRUE	TRUE	TRUE	TRUE	FALSE	FALSE		astrophysics
-	astroFacility	Facility	The observatory or facility where the data was obtained. 		text	1		TRUE	TRUE	TRUE	TRUE	FALSE	FALSE		astrophysics
```

Comment:
I assume these relate to the todo? What is the reason for changing these values? (I probably just missed it, now I'm just curious)

Reply:
Not related to the todo. I think these are just inconsistencies - the block says they are controlled vocab fields but there are no vocab values for them in the block.

Reply:
That makes sense - I just wonder how why they were ever true in the first place?